packages/truffle/test/scenarios/commands/migrate.js

```diff
@@ -19,7 +25,13 @@ describe("truffle migrate", () => {
 
   describe("when run on the most basic truffle project", () => {
     it("doesn't throw", async () => {
-      await CommandRunner.run("migrate", config);
+      try {
+        await CommandRunner.run("migrate", config);
+      } catch (error) {
+        console.log("the logged stuff -- %o", config.logger.loggedStuff);
```

Comment:
Did you mean to leave this? If so I'd probably leave a more descriptive message in the logging? What is config.logger.loggedStuff usually conveying?